app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -206,17 +207,184 @@ class StateFragmentTest {
   //     answers for each of the interactions.
   //  2. Verifying the button visibility state based on whether text is missing, then
   //     present/missing for text input or numeric input.
+
   //  3. Testing providing the wrong answer and showing feedback and the same question again.
+  @Test
+  fun testStateFragment_loadExp_secondState_submitWrongAnswer_feedbackAndQuestionVisible() {
+    launchForExploration(TEST_EXPLORATION_ID_2).use {
+      startPlayingExploration()
+      clickContinueInteractionButton()
+
+      // Attempt to submit an wrong answer.
+      typeFractionText("1/3")
+
+      clickSubmitAnswerButton()
+
+      scrollToViewType(FEEDBACK)
+      onView(withId(R.id.feedback_text_view)).check(matches(isDisplayed()))
+      onView(withId(R.id.fraction_input_interaction_view)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testStateFragment_loadExp_secondState_submitWrongAnswerTwice_feedbackAndQuestionVisible() {
+    launchForExploration(TEST_EXPLORATION_ID_2).use {
+      startPlayingExploration()
+      clickContinueInteractionButton()
+
+      // Attempt to submit an wrong answer.
+      typeFractionText("1/3")
+      clickSubmitAnswerButton()
+
+      // Attempt to submit an wrong answer twice.
+      typeFractionText("1/9")
+      clickSubmitAnswerButton()
+
+      scrollToViewType(FEEDBACK)
+      onView(withId(R.id.feedback_text_view)).check(matches(isDisplayed()))
+      onView(withId(R.id.fraction_input_interaction_view)).check(matches(isDisplayed()))
+    }
+  }
+
   //  4. Configuration change with typed text (e.g. for numeric or text input) retains that
   //     temporary
   //     text and you can continue with the exploration after rotating.
+
+  @Test
+  @Ignore("Work on issue regarding this test is in progress")
```

Comment:
Do we need a discussion on this?

Reply:
This test is regarding retaining text in EditText during config change, currently @FareesHussain is working on that: https://github.com/oppia/oppia-android/pull/2514

Reply:
in this case we need to skip testing this part. 
skip
```
testStateFragment_loadExp_submitAnswer_configChange_explorationContinues
```